frontend/src/liboperator/api/v1/messages.go

```diff
@@ -187,6 +187,8 @@ type CVD struct {
 	Displays []string `json:"displays"`
 	// [Output Only]
 	WebRTCDeviceID string `json:"webrtc_device_id"`
+	// [Output Only]
+	ADBPort int `json:"adb_port"`
```

Comment:
why not return the original `ADB serial` and get rid of the parsing logic and error handling here?

Reply:
What's the reason why we should move parsing code to the client side? Is making same with `cvd fleet` the purpose of `GET /cvds`?

Reply:
not exactly the same but as closer as possible. in this case, if `cvd fleet` returns empty or misspelled  `ADBSerial` values, the host orchestrator should provide the same values to clients rather than failing. clients can  then raise more relevant errors to the final users.

we should see the host orchestrator as an http wrapper of `cvd` given clients the same values presented in `cvd` outputs.

Reply:
Okay. Then I'll just pass `ADBSerial` value in HO, and handle it in client side like `cvdr` binary. Done in https://github.com/google/android-cuttlefish/pull/522, by changing the field from `ADBPort` to `ADBSerial`.